app/airq/commands/quality.py

```diff
@@ -22,7 +26,9 @@ def handle(self) -> typing.List[str]:
 
         if not zipcode.pm25 or zipcode.is_pm25_stale:
             return [
-                f'Oops! We couldn\'t determine the air quality for "{zipcode.zipcode}". Please try a different zip code.'
+                'Oops! We couldn\'t determine the air quality for "{}". Please try a different zip code.'.format(
```

Comment:
Why switch to a `.format` call?

Reply:
f' strings aren't nice for when we start wrapping strings with `gettext()`, we can also change to the other format you mentioned.

Reply:
for gettext we need to use the `%s` format because they don't understand format or fstrings. But this is a good step in that direction so lgtm

Reply:
We can use format because it returns the value before the call to format